server/app.ts

```diff
@@ -0,0 +1,63 @@
+import * as fs from "fs";
+import * as path from "path";
+import * as express from "express";
+import * as compression from "compression";
+import * as bodyParser from "body-parser";
+
+import { graphqlExpress, graphiqlExpress } from 'graphql-server-express';
+import { makeExecutableSchema } from 'graphql-tools';
+
+import { PORT } from './common';
+import * as plugins from './plugins/api';
+import { GenericNotifier } from './plugins/api/GenericNotifier';
+import { APIReturn } from './plugins/api/APIReturn';
+
+const typeDefs = fs.readFileSync(path.resolve(__dirname, "../api.graphql"), "utf8");
+
+const app = express();
+
+app.use(compression());
+process.on("unhandledRejection", err => {
+	throw err;
+});
+
+interface IStatusReturn {
+	[medium: string]: [Promise<APIReturn>] | APIReturn; // String for generic unpacking error
+}
+
+const resolvers = {
+	Query: {
+		send_message: (prev: any, args: any) => {
+			let statusRet: IStatusReturn = {};
+			const src = Object.keys(args.plugins);
+			src.forEach( name => {
```

Comment:
Should run "concurrently" by making a list of promises and then awaiting on the entire list.
But this should be preceeded by the same thing with `check`.
This would make the notifications go faster